presto-main/src/main/java/com/facebook/presto/sql/gen/BytecodeExpressionVisitor.java

```diff
@@ -119,8 +119,7 @@ public BytecodeNode visitCall(CallExpression call, final Scope scope)
                     generator = new RowConstructorCodeGenerator();
                     break;
                 case BIND:
-                    generator = new BindCodeGenerator();
-                    break;
+                    throw new UnsupportedOperationException();
```

Comment:
I use this as a placeholder for now. We can discuss what's the convention in Presto to handle such cases. (I see both `UnsupportedOperationException` and `checkCondition` are used).